### ISSUE TITLE
[Anon] Bug report: Protectorate_of_Menoth_Reckoning(2015).catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/warmahordes#46]
**File:** Protectorate_of_Menoth_Reckoning(2015).catz

**BattleScribe version:** 1.15.07

**Platform:** Windows

**Dropbox:** Yes

**Description:** The unit Flame Bringers is not available, like the mercenaries 's solo Rutger Shaw.

Thank's you

**Comment by Obiken** [comment]
Verified. Flame Bringers and Rutger Shaw is missing in Protectorate. Rutger also missing in Mercenaries.
I've currently packed my books for a good while, so hope someone else can fix this.

**Comment by JimOrtiz** [comment]
I show "Rutger Shaw, Professional Adventurer" in the Mercenaries catalog XML in Git.  I'm away from my BattleScribe editors at the moment, I wonder if he's not properly tagged as "Mercenary (Protectorate)" and the tool for copying Mercs into the other catalogs isn't catching him.

I did notice that the last check-in for Protectorate mentions "No Mercenaries yet." for the Reckoning additions.  I can't remember what book Rutger is in...maybe he was added to Mercs, but the other catalogs weren't updated to include the new Mercs?

  Out of curiosity, I'd like to know if the Flame Bringers were in lost when I added Reckoning, or if they've always been missing, but it doesn't really matter.
